sdk/go/api_esc_extensions.go

```diff
@@ -0,0 +1,287 @@
+package esc
```

Comment:
Nit: this package name will conflict with the name of the root `esc` package. That might be fine--just wanted to call it out.